src/query/service/tests/it/storages/testdata/system-tables.txt

```diff
@@ -348,6 +349,7 @@ DB.Table: 'system'.'settings', Table: settings-table_id:1, ver:0, Engine: System
 | load_file_metadata_expire_hours | 168        | 168         | SESSION | How many hours will the COPY file metadata expired in the metasrv, default value: 24*7=7days                                                                           | UInt64 |
 | max_block_size                  | 65536      | 65536       | SESSION | Maximum block size for reading, default value: 65536.                                                                                                                  | UInt64 |
 | max_execute_time                | 0          | 0           | SESSION | The maximum query execution time. it means no limit if the value is zero. default value: 0.                                                                            | UInt64 |
+| max_memory_usage                | 1073741824 | 27487790694 | SESSION | The maximum memory usage for processing single query, in bytes. By default the value is determined automatically.                                                      | UInt64 |
```

Comment:
Here should be `max_server_memory_usage`.

Reply:
This is settings table.